.github/workflows/ansible-test-plugins.yml

```diff
@@ -58,14 +58,14 @@ jobs:
         db_engine_version:
           - mysql_5.7.31
           - mysql_8.0.22
-          - mariadb_10.2.37
+          - mariadb_10.3.34
           - mariadb_10.5.9
         ansible:
           - stable-2.9
           - stable-2.10
           - stable-2.11
           - stable-2.12
-          #- devel
+          - devel
```

Comment:
What's the purpose of devel branch? I ask because since I'm in the project we never used a devel branch, only main.

Reply:
this is the current devel branch of ansible-core. Shortly `stable-2.13` will be branched from it and if our code doesn't work with `devel` now, it will not work with the `stable-2.13` as well, i.e. it won't work with ansible-core 2.13 and Ansible 6 respectively. So this is why i consider important to test against it at least some time before the next ansible release (May, IIRC)